### PR TITLE
Add rocstar price automation demo

### PR DESCRIPTION
**Pull request by imstar15**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,11 +20,11 @@
...
     "@mangata-finance/sdk": "^1.20.1",
-    "@oak-network/adapter": "^0.1.3",
+    "@oak-network/adapter": "0.1.0-rc.21",
     "@oak-network/api-augment": "^2.0.7",
-    "@oak-network/config": "^0.1.2",
-    "@oak-network/sdk": "^0.1.3",
-    "@oak-network/sdk-types": "^0.1.3",
+    "@oak-network/config": "0.1.0-rc.21",
+    "@oak-network/sdk": "0.1.0-rc.21",
+    "@oak-network/sdk-types": "0.1.0-rc.21",
```
Comment (chrisli30) [inline package.json:27]: Please completely get rid of this `rc.21` format. The number plus string format is hard for version comparison. Use the Github Action to publish a dev version!

File: src/astar/shibuya-price.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Keyring from '@polkadot/keyring';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import { chains } from '@oak-network/config';
+import { schedulePriceTask } from './price-common';
+
+const createTaskPayload = (astarApi) => astarApi.tx.system.remarkWithEvent('Hello world!');
```
Comment (chrisli30) [inline src/astar/shibuya-price.js:6]: Please change this to the ethereumChecked.transaction extrinsic even if the proxyExecuted will fail. We should stop using the `system.remarkWithEvent`, which is never the real use case. For now, we only need to make sure we see the xcmQueue event on the other parachain.

**Review by chrisli30 — CHANGES_REQUESTED** [review]

File: src/astar/shibuya-price.js (hunk 2 above)
Comment (imstar15) [reply src/astar/shibuya-price.js]: Fixed

File: package.json (hunk 1 above)
Comment (imstar15) [reply package.json]: Fixed

**Review by chrisli30 — APPROVED** [review]
The code looks good 👌

**Pull request merged**